test.md

```diff
@@ -0,0 +1,318 @@
+---
+description: Smoketest page
```

Comment:
Drafts are for "posts" -- as in, blog posts. We only use pages, which don't have draft functionality.